### PR TITLE
Feat/lens commands

### PR DESCRIPTION
**Pull request by Kenny-Gin1**
image

### CODE REVIEW

File: apps/web/src/components/commands/social/lens/Lens.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <DataPill query={query} className="bg-green-100">
+      <Link
+        href={`https://lenster.xyz/u/${handle.replace(".lens", "")}`}
+        className="no-underline"
+        target="_blank"
+      >
+        🌿
+      </Link>{" "}
+      {query.data?.name ? `${query.data?.name} (${handle})` : handle}
```
Comment (ericzakariasson) [inline apps/web/src/components/commands/social/lens/Lens.tsx:40]: Should we use the `Profile` from the widgets here as well? had an idea for the designs
image

**Review by ericzakariasson — APPROVED** [review]
great job! 👏

File: apps/web/src/components/commands/social/lens/command.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { LensWidgetProps } from "./Lens";
+
+
+export const lensProfileCommand: CommandConfiguration<LensWidgetProps> = {
+  command: "lens-profile",
+  title: "Lens Profile",
+  description: "Lens statistics for a profile",
+  icon,
+
+  blockType: "inline",
```
Comment (philipandersson) [inline apps/web/src/components/commands/social/lens/command.ts:13]: Is the profile really inline or should it be a block? @Kenny-Gin1 @ericzakariasson

**Review by philipandersson — APPROVED** [review]

File: apps/web/src/components/commands/social/lens/command.ts (hunk 2 above)
Comment (ericzakariasson) [reply apps/web/src/components/commands/social/lens/command.ts:13]: That is true, should probably be a block!

**Pull request merged**